relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py

```diff
@@ -118,6 +118,15 @@ def dictToAsciiTable(data):
            break
     return outmsg
 
+
+def check_url():
```

Comment:
this does not seem to be used anywhere

Reply:
/me adds a file that was not commited 

Sure ? can you look better please ? i would not have forgotten a file ...